### ISSUE TITLE
API Documentation when?!

### ISSUE THREAD

**Issue by binn** [Nekos-life/neko-website#2]
Proper documentation please.

**Comment by 31ank** [comment]
you can get most infos from https://nekos.life/api/v2/endpoints

**Comment by binn** [comment]
That's not documentation, sadly.

**Comment by 31ank** [comment]
True, but still better than nothing. You can use most functions when you read the endpoints json and the code of the apis. Thats how I got my program to work with nekos.life
